types/config.go

```diff
@@ -0,0 +1,8 @@
+package types
+
+import "github.com/ethereum/go-ethereum/common"
+
+type ElChainReaderConfig struct {
+	DelegationManagerAddress common.Address
+	AvsDirectoryAddress      common.Address
+}
```

Comment:
think this should just be in the elchainreader module?

Reply:
yea I am using this in utils bindings too - putting their creates circular binding I believe.

Reply:
let's folow https://go.dev/blog/package-names and stop using utils modules everywhere?
And let's move bindings.go under elcontracts/bindings.go so that they are in the same packge?

Reply:
Done

Reply:
config should be moved from types/config.go to elcontracts/reader.go no?

Reply:
oh shit. did I not do it. me bad. will do

Reply:
moved b9ba1bed85f61239f75c1c5b3d917c30ba4a9553